lib/features/rules/ChoreoRules.js

```diff
@@ -197,10 +196,17 @@ ChoreoRules.prototype.init = function() {
         source = context.source,
         target = context.target;
 
+
+    // Check if is either participant or message from choreo
+    // Todo: more fine grained implementation based on target, e.g should't be possible to paste only band
```

Comment:
What's up with this? I still get console errors when I try to copy/paste a band only, and afterwards some things like resizing do not work anymore.

Reply:
Should we create  a new issue for this and solve it separately?